pep-0649.rst

```diff
@@ -1,13 +1,14 @@
 PEP: 649
 Title: Deferred Evaluation Of Annotations Using Descriptors
 Author: Larry Hastings <larry@hastings.org>
-Status: Draft
+Status: Accepted
```

Comment:
Pending SC review — please mark this comment as resolved IFF and when the SC accepts the changes (so it blocks accidental merge otherwise). Also marked as `DO-NOT-MERGE` (which is currently not actually checked by the CIs on this repo; Draft would be a more robust solution but could send the wrong signal about others reviewing it).